src/Component/Symbolizer/Preview/Preview.spec.tsx

```diff
@@ -13,11 +13,12 @@ describe('Preview', () => {
   }];
 
   beforeEach(() => {
-    wrapper = TestUtil.shallowRenderComponent(Preview, {
+    const props: PreviewProps = {
       locale: en_US.GsPreview,
-      onSymbolizerChange: jest.fn(),
+      onSymbolizersChange: jest.fn(),
```

Comment:
Why not declaring `let onSymbolizersChangeDummy: jest.Mock;` and assigning it to `onSymbolizersChange`? Would be consistent with other tests and easier to spy on, if tests will be extended.